layers/+lang/ipython-notebook/packages.el

```diff
@@ -157,4 +158,11 @@
         ("9" ein:notebook-worksheet-open-last)
         ("+" ein:notebook-worksheet-insert-next)
         ("-" ein:notebook-worksheet-delete)
-        ("x" ein:notebook-close)))))
+        ("x" ein:notebook-close))
+      (spacemacs/set-leader-keys "ein" 'spacemacs/ipython-notebook-transient-state/body))))
+
+(defun ipython-notebook/init-ob-ipython ()
+  (use-package ob-ipython
+    :defer t
```

Comment:
I think `:defer t` causes it to not be loaded upon start time but when the layer is loaded. This worked on my setup, isn't it working anymore?